clients/datasetapi.go

```diff
@@ -0,0 +1,17 @@
+package clients
```

Comment:
Because you have created this package, maybe it would make sense to move the zebedee client into here too?

Reply:
Fixed in update 1.